cfpb/consumerfinance.gov
# Add hook to work with income_level_group data shape

## Additions

- Added a hook to work with income_level_group data shape

## How to test this PR

1. Run locally
2. Go to http://localhost:8000/admin/pages/4511/edit/
3. Add a simple-chart block using data source `https://files.consumerfinance.gov/data/consumer-credit-trends/mortgages/volume_data_Income_Level_MTG.csv`
4. Add `cct_income` into Transform field
5. Publish and view live
6. Note that the chart can read and display the data
